src/assets/translations/en-US.json

```
@@ -740,17 +746,20 @@
       "Survey": "Survey",
       "Suspend": "Suspend",
       "Take Survey": "Take Survey",
+      "Test Connection": "Test Connection",
       "Transaction Reverted": "Transaction Reverted",
       "Transfer Client": "Transfer Client",
       "Transfer Clients": "Transfer Clients",
       "Unassign Staff": "Unassign Staff",
       "Undo": "Undo",
       "Undo Rejection": "Undo Rejection",
       "Undo Transfer": "Undo Transfer",
+      "undo_account_transfer": "Undo Account Transfer",
```

Biome (2.5.3)

[error] The key undo_account_transfer was already declared.

(lint/suspicious/noDuplicateObjectKeys)